pkg/cluster/resources.go

```diff
@@ -154,6 +154,30 @@ func CreateNamespace(ctx context.Context, cli client.Client, namespace string, m
 	return desiredNamespace, client.IgnoreAlreadyExists(createErr)
 }
 
+// ProcessAllNamespace execute func() on all active namespaces in the cluster with 500 as chunksize.
+func ProcessAllNamespace(ctx context.Context, cli client.Client, processFunc func(*corev1.Namespace) error) error {
```

Comment:
Didn't occur to me before, and naming is hard, but perhaps `ExecuteOnAllNamespaces` is a bit more self-explanatory.

I would also skip the 500 chunk size in the godoc. This can be changed in the code later and left behind in the godoc causing confusion. It is also not that relevant to document in my eyes.

Reply:
I would use `ForEachNamespace`, https://en.wikipedia.org/wiki/Foreach_loop